app/assets/javascripts/discourse/app/components/user-fields/text.hbs

```diff
@@ -1,13 +1,16 @@
 <div class="controls">
-  <label class="control-label" for={{concat "user-" this.elementId}}>
-    {{this.field.name}}
-    {{#if this.field.required}}<span class="required">*</span>{{/if}}
-  </label>
   <Input
     id={{concat "user-" this.elementId}}
     @value={{this.value}}
     maxlength={{this.site.user_field_max_length}}
   />
+  <label
+    class="control-label alt-placeholder"
+    for={{concat "user-" this.elementId}}
+  >
+    {{this.field.name}}
+    {{#if this.field.required}}<span class="required">*</span>{{/if}}
+  </label>
```

Comment:
This is after the input in the DOM but styling changes things so it's placed as a placeholder, then moved on focus, right?

Reply:
Yep that is correct.